gr-blocks/lib/interleaved_short_to_complex_impl.cc

```diff
@@ -43,9 +46,10 @@ int interleaved_short_to_complex_impl::work(int noutput_items,
                                             gr_vector_void_star& output_items)
 {
     const short* in = (const short*)input_items[0];
-    gr_complex* out = (gr_complex*)output_items[0];
+    float* out = (float*)output_items[0];
 
-    interleaved_short_array_to_complex(in, out, 2 * noutput_items);
+    // This calculates in[] * 1.0 / d_scalar
+    volk_16i_s32f_convert_32f_u(out, in, d_scalar, 2 * noutput_items);
```

Comment:
Nice use of Volk! I'm not sure you have to specify `_u` for the unaligned here. Can you do benchmarks on these, to see if there's any difference between adding `_u` and not?